File: Feitos em C/2803.c
#include <stdio.h>
int main() {
	char regiao[30];
    scanf("%s",&regiao);
    if(strcmp(regiao,"para")==0||strcmp(regiao,"roraima")==0||strcmp(regiao,"acre")==0||strcmp(regiao,"amapa")==0||strcmp(regiao,"amazonas")==0||strcmp(regiao,"rondonia")==0||strcmp(regiao,"tocantins")==0){
    	printf("Regiao Norte\n");
	}
	else{
		printf("Outra regiao\n");
	}
    return 0;
}

File: Feitos em C Sharp/2803.cs
using System;
namespace ConsoleApp16 {
    class Program {
        static void Main(string[] args) {
            string regiao = Console.ReadLine();
            if (regiao == "para" || regiao == "roraima" || regiao == "acre" || regiao == "amapa" || regiao == "amazonas" || regiao == "rondonia" || regiao == "tocantins") {
                Console.WriteLine("Regiao Norte");
            }
            else {
                Console.WriteLine("Outra regiao");
            }
        }
    }
}
